### DISCUSSION TITLE
dns servers

### DISCUSSION THREAD

**Discussion by typicallol** [zek-c/Securly-Kill-V111#73] [General]
ok so since the dns server is hosted on omadadns can someone get like another dns server in case one fails or tell us how u wrote the code to use the dns server and then we can add a lot more extensions to the code

**Comment by dragon731012** [comment]
Just read it

**Reply by typicallol** [reply]
i aint that smart just tell us generally in idiot language how it actually works

**Reply by dragon731012** [reply]
![image](https://github.com/zek-c/Securly-Kill-V111/assets/105754685/55b488b0-d416-4876-a987-d9ee91c0b476)

**Reply by dragon731012** [reply]
Edit that

**Reply by typicallol** [reply]
so i just get a server and change it thats all?

**Reply by dragon731012** [reply]
Yes

**Reply by typicallol** [reply]
damn im stupid thats ez i just find a server
